app/controllers/application_controller.rb

```diff
@@ -1,10 +1,79 @@
+require 'pry'
 require './config/environment'
 
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "evelynwaugh"
   end
 
-end
\ No newline at end of file
+  helpers do
+    def logged_in?
+      if session[:user_id]
+        true
+      else
+        false
+      end
+    end
+
+  end
+
+  get '/' do
+    erb :'home/index'
+  end
+
+  get '/login' do
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller).